### PR TITLE
.45 lethal and practice rounds changes

### PR DESCRIPTION
**Pull request by SonixApache**
Knocking someone to the floor made follow-up magdumping a little too easy.
[Webm](https://puu.sh/u1nJK/c446ba2d44.webm)
:cl:
 - tweak: .45 lethals now weaken rather than stun
 - tweak: .45 practice rounds no longer stun

### CODE REVIEW

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -115,16 +115,17 @@
 	weaken = 0
 
 /obj/item/projectile/bullet/fourtyfive //secgun ammo!
-	damage = 15
-	stun = 2 //stopping power
-	weaken = 2
+	damage = 13
+	drowsy = 0.25 //stopping power
```
Comment (Sprok0) [inline code/modules/projectiles/projectile/bullets.dm]: 0.25 seems incredibly tiny, drowsyness is reduced by 1 every life tick. And agony on projectiles deals hallucination damage, 0.25 of that might as well not exist

File: code/modules/projectiles/projectile/bullets.dm [hunk 2]
```diff
@@ -115,16 +115,17 @@
...
 	penetration = 2
 
 /obj/item/projectile/bullet/fourtyfive/practice
 	damage = 2
-	stun = 1
-	weaken = 1
+	drowsy = 0
+	agony = 0
 	embed = 0
+	penetration = 0
```
Comment (Intigracy) [inline code/modules/projectiles/projectile/bullets.dm:128]: You should make rubber bullets not penetrate.

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (Intigracy) [reply code/modules/projectiles/projectile/bullets.dm]: Just leave drowsy/agony at 0 like stun/weaken, like most every other lethal projectile.

File: code/modules/projectiles/projectile/bullets.dm (hunk 2 above)
Comment (Shadowmech88) [reply code/modules/projectiles/projectile/bullets.dm:128]: Wouldn't that be non-atomic?

Comment (Shadowmech88) [reply code/modules/projectiles/projectile/bullets.dm:128]: Nevermind, I see that you're talking about a child of `/obj/item/projectile/bullet/fourtyfive`.

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (SonixApache) [reply code/modules/projectiles/projectile/bullets.dm]: Agony seemed to cause pain / slumping in testing. Drowsy's there to slow down the target a bit. It's one tick of drowsy, which is enough. I could just up it to one.

File: code/modules/projectiles/projectile/bullets.dm (hunk 2 above)
Comment (SonixApache) [reply code/modules/projectiles/projectile/bullets.dm:128]: done

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (Intigracy) [reply code/modules/projectiles/projectile/bullets.dm]: That'll work.

**Pull request merged**